ess-dmsc/nexus-constructor
# Adding OFF renderer tests, refactoring off_renderer

### Issue

None

### Description of work

Worked with Dom on Friday to split up some of the big system-test style tests into unit tests, and in doing this we split up most of the OFF renderer code into small functions with single responsibilities. 

### Acceptance Criteria 
To test these changes: 
- Make sure the sample gets loaded properly with no error codes in the logger. 
- Load a .off file with a few different component types and see if the display gets updated with the model. 

### Nominate for Group Code Review

- [ ] Nominate for code review
